podcasts/DiscoverViewController.swift

```diff
@@ -83,7 +83,7 @@ class DiscoverViewController: PCViewController {
         guard let index = notification.object as? Int, index == tabBarItem.tag else { return }
 
         let defaultOffset = -PCSearchBarController.defaultHeight - view.safeAreaInsets.top
-        if mainScrollView.contentOffset.y > defaultOffset {
+        if mainScrollView.contentOffset.y.rounded() > defaultOffset.rounded() {
```

Comment:
My first reaction was that we simply don't care about the fractional value, so maybe rounding [`down`](https://developer.apple.com/documentation/swift/floatingpointroundingrule/down) or simply casting to `Int` would make more sense?

In using `rounded` here, I believe the default rounding method is [toNearestOrAwayFromAero](https://developer.apple.com/documentation/swift/floatingpointroundingrule/tonearestorawayfromzero) and I just wonder if we would ever run into a case where a > `.5` value might be compared with < `.4` and we really don't care about that.

Here are the examples:
```
(5.2).rounded(.toNearestOrAwayFromZero)
// 5.0
(5.5).rounded(.toNearestOrAwayFromZero)
// 6.0
(-5.2).rounded(.toNearestOrAwayFromZero)
// -5.0
(-5.5).rounded(.toNearestOrAwayFromZero)
// -6.0
```

I think this is probably very nitpicky so take it or leave it – just my initial reaction.

Reply:
That's a good point! Changed in dfe88068e62e6b09290f9bde91cd5ab8de974ee9